code/headers/dummy_controller_c.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <controller_interface_c.hpp>
+
+namespace r2d2::manual_control {
+    
+    
+
+    class dummy_controller_c: public controller_interface_c{
+    private:
+    //initializing all buttons, sliders and joysticks at their base (0)
+    bool buttons[12] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Is it not cleaner to use:

```cpp
bool buttons[12] = {};
```
?

Reply:
I didn't know that initializing like that garanties that all the values are 0. I will change this in the next commit